torch/ao/quantization/fx/graph_module.py

```diff
@@ -89,7 +89,11 @@ class QuantizedGraphModule(GraphModule):
     """
     def __init__(self, root: Union[torch.nn.Module, Dict[str, Any]], graph: Graph, preserved_attr_names: Set[str]):
         self.preserved_attr_names = preserved_attr_names
-        preserved_attrs = {attr: getattr(root, attr) for attr in self.preserved_attr_names if hasattr(root, attr)}
+        preserved_attrs = {
```

Comment:
when is `attr` not string?

Reply:
this is a bit weird since preserved_attr_names should be passed from users and it should be a set of `str` I think

Reply:
PNP has some weird cases where weight packing happens on the fly

Reply:
I see, should we just error out if user provides non str in `preserved_attr_names`?

Reply:
yeah, that's current behavior, I'm going to fix this elsewhere, I think you're right